tests/unit/components/item/ItemHero.spec.js

```diff
@@ -83,87 +83,87 @@ const identifier = '/2020601/https___1914_1918_europeana_eu_contributions_10265'
 
 describe('components/item/ItemHero', () => {
   describe('selectMedia', () => {
-    context('when a new item is selected', () => {
+    describe('when a new item is selected', () => {
       it('updates the identifier', () => {
         const wrapper = factory({ media, identifier });
         wrapper.vm.selectMedia(media[1].about);
-        wrapper.vm.selectedMedia.about.should.eq(media[1].about);
+        expect(wrapper.vm.selectedMedia.about).toBe(media[1].about);
       });
       it('updates the rights statement', () => {
         const wrapper = factory({ media, identifier });
         wrapper.vm.selectMedia(media[1].about);
-        wrapper.vm.selectedMedia.webResourceEdmRights.def[0].should.eq(media[1].webResourceEdmRights.def[0]);
+        expect(wrapper.vm.selectedMedia.webResourceEdmRights.def[0]).toBe(media[1].webResourceEdmRights.def[0]);
       });
       it('unsets any selected IIIF canvas', async() => {
         const wrapper = factory({ media, identifier });
         await wrapper.setData({ selectedCanvas: { about: 'http://www.example.org/canvas' } });
         wrapper.vm.selectMedia(media[1].about);
-        (wrapper.vm.selectedCanvas === null).should.eq(true);
+        expect(wrapper.vm.selectedCanvas === null);
       });
     });
   });
 
   describe('downloadEnabled', () => {
-    context('when the rightsstatement is in copyright', () => {
+    describe('when the rightsstatement is in copyright', () => {
       it('is false', () => {
         const wrapper = factory({ media: [media[1]], identifier });
-        wrapper.vm.downloadEnabled.should.eq(false);
+        expect(wrapper.vm.downloadEnabled).toBe(false);
       });
     });
-    context('when the selected media is the isShownAt and not downloadable', () => {
+    describe('when the selected media is the isShownAt and not downloadable', () => {
       it('is false', () => {
         const wrapper = factory({ media: [media[5]], identifier });
-        wrapper.vm.downloadEnabled.should.eq(false);
+        expect(wrapper.vm.downloadEnabled).toBe(false);
       });
     });
-    context('when the rightsstatement is not in copyright and the selected media is not the isShownAt', () => {
+    describe('when the rightsstatement is not in copyright and the selected media is not the isShownAt', () => {
       it('is true', () => {
         const wrapper = factory({ media: [media[0]], identifier });
-        wrapper.vm.downloadEnabled.should.eq(true);
+        expect(wrapper.vm.downloadEnabled);
```

Comment:
This won't fail for me even when downloadEnabled returns false.

It needs to be `expect(wrapper.vm.downloadEnabled).toBe(true);`

Reply:
The same applies elsewhere where there is no additional criteria after the `expect(x)` part.

A notable example that's quite common are a lot of the `nock.isDone()` calls. Although it's likely that those still fail if actuall unexpect requests are made which are otherwise not registered in Nock.